src/main/java/me/botsko/prism/commands/DebugCommand.java

```diff
@@ -30,8 +30,21 @@
 
     @Override
     public void handle(CallInfo call) {
+        if(call.getArgs().length == 1) {
+            String arg = call.getArg(0);
+            switch (arg.toLowerCase()){
```

Comment:
Codacy found an issue: [WhitespaceAround: '{' is not preceded with whitespace.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)